sdk/trace/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -370,7 +368,7 @@ public ReadWriteSpan setStatus(StatusCode statusCode, @Nullable String descripti
 
   @Override
   public ReadWriteSpan recordException(Throwable exception) {
-    recordException(exception, null);
+    recordException(exception, Attributes.empty());
```

Comment:
Alternative is to extract a private method with a Nullable argument, but this seemed fine to me